### PR TITLE
add v1.0 release notes

### PR DESCRIPTION
**Pull request by kevinintel**

### CODE REVIEW

**Review by dbkinder — APPROVED** [review]
No complaints from me

File: release_notes/v1.0.md [hunk 1]
```diff
@@ -0,0 +1,450 @@
...
+    - Optimize the microservice launch and remove the dispatch overhead
+    - Enable HorizontalPodAutoscaler (HPA) for better resource management
+    - Define the metrics of RAG performance and enable accuracy evaluation for more GenAI examples
+    - Further improvement on documentation and developer experience 
+
+- Other features
+    - Enable OpenAI compatible format on new microservices
+    - Support microservice launch from ModelScope to address China ecosystem need
+    - Support Red Hat OpenShift Container Platform (RHOCP)
+    - Refactor the code and CI/CD pipeline to provide better support for external contributors
```
Comment (hshen14) [inline release_notes/v1.0.md:18]: remove external

File: release_notes/v1.0.md [hunk 2]
```diff
@@ -0,0 +1,450 @@
...
+    - Improve Docker versioning to avoid the conflict
```
Comment (hshen14) [inline release_notes/v1.0.md:19]: potential conflict

File: release_notes/v1.0.md [hunk 3]
```diff
@@ -0,0 +1,450 @@
...
+    - Enable OpenAI compatible format on new microservices
+    - Support microservice launch from ModelScope to address China ecosystem need
+    - Support Red Hat OpenShift Container Platform (RHOCP)
+    - Refactor the code and CI/CD pipeline to provide better support for external contributors
+    - Improve Docker versioning to avoid the conflict 
+
+- Learn more about OPEA at
+    - Getting Started: https://opea-project.github.io/latest/index.html
+    - Github: https://github.com/opea-project
+    - GitHub IO: https://opea-project.github.io/latest/index.html
```
Comment (hshen14) [inline release_notes/v1.0.md:24]: Remove this line, as same as the Getting Started

File: release_notes/v1.0.md [hunk 4]
```diff
@@ -0,0 +1,450 @@
...
+    - Docker Hub: https://hub.docker.com/u/opea
+
+- Release Documentation:
+    - Landing page:  https://opea.dev/
```
Comment (hshen14) [inline release_notes/v1.0.md:28]: Page

File: release_notes/v1.0.md [hunk 5]
```diff
@@ -0,0 +1,450 @@
...
+    - Provide the experimental LLM model training support including full fine-tuning, and parameter-efficient fine-tuning (PEFT)
+    - Improve RAG with Knowledge Graph based on Neo4j
+    - Improve VisualQnA and provide multi-modality RAG support
+    - Optimize the microservice launch and remove the dispatch overhead
+    - Enable HorizontalPodAutoscaler (HPA) for better resource management
+    - Define the metrics of RAG performance and enable accuracy evaluation for more GenAI examples
+    - Further improvement on documentation and developer experience 
+
+- Other features
+    - Enable OpenAI compatible format on new microservices
```
Comment (ftian1) [inline release_notes/v1.0.md:15]: This description is not accurate. Not all new added microservices can be compatible with openAI. we just enabled those microservices who can be compatible with openAI.

File: release_notes/v1.0.md (hunk 2 above)
Comment (ftian1) [reply release_notes/v1.0.md:19]: naming convention vs versioning?

**Review by mkbhanda — COMMENTED** [review]
some edits

**Review by hshen14 — APPROVED** [review]

File: release_notes/v1.0.md [hunk 6]
```diff
@@ -0,0 +1,450 @@
+# OPEA Release Notes v1.0
+## What’s New in OPEA v1.0
+
+- Highlights
+    - Improve the RAG performance with microservice optimization (e.g., Hugging Face TGI, vLLM) and megaservice tuning
+    - Provide the experimental LLM model training support including full fine-tuning, and parameter-efficient fine-tuning (PEFT)
+    - Improve RAG with Knowledge Graph based on Neo4j
+    - Improve VisualQnA and provide multi-modality RAG support
+    - Optimize the microservice launch and remove the dispatch overhead
+    - Enable HorizontalPodAutoscaler (HPA) for better resource management
```
[the remark this answers is not in the document]
Comment (kevinintel) [reply release_notes/v1.0.md:10]: last release we claimed "GenAI Microservices Connector (GMC) successfully tested on Nvidia GPUs", should we point out Nvidia again?

**Comment by mkbhanda** [comment]
@kevinintel - Surprised that my edits indicating improvements to GMC, helm charts for Nginx with CORs etc not merged

**Comment by ftian1** [comment]
> @kevinintel - Surprised that my edits indicating improvements to GMC, helm charts for Nginx with CORs etc not merged

Sorry, Malini. When I reviewed this line, I thought it's better to change to such words "integrate nginx with CORS protection and data preparation". My point here is we need introduce what features are introduced but not detail implementation way like Helm Charts. If you think this is not quite accurate, I can submit PR to fix it.

**Pull request merged**